### PR TITLE
LPS-188177 [CP 2.0] Set up the e-mail structure

### PR DESCRIPTION
**Pull request by pedroliveirav**
[LPS-188177](https://liferay.atlassian.net/browse/LPS-188177)

### CODE REVIEW

**Review by kevenleone — CHANGES_REQUESTED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/graphql/queries.js [hunk 1]
```diff
@@ -616,3 +616,36 @@ export const deleteAccountUserAccount = gql`
...
+		notificationTemplateByExternalReferenceCode(
+			externalReferenceCode: $externalReferenceCode
+		) {
+			body
+			dateCreated
+			id
+			name
+			recipients
+			subject
+			type
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/graphql/queries.js:660]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/graphql/queries.js [hunk 2]
```diff
@@ -616,3 +616,36 @@ export const deleteAccountUserAccount = gql`
 		)
 	}
 `;
+
+export const getTemplateByExternalReferenceCode = gql`
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/services/liferay/graphql/queries.js]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/index.jsx [hunk 3]
```diff
@@ -216,6 +221,99 @@ const SetupDXPCloudPage = ({
...
+				for (const key in emailDataList) {
+					if (emailDataList.hasOwnProperty(key)) {
+						const placeholder = key;
+						const value = emailDataList[key];
+						formattedEmailSubject = formattedEmailSubject.replace(
+							placeholder,
+							value
+						);
+					}
+				}
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/index.jsx]: Done.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/index.jsx [hunk 4]
```diff
@@ -216,6 +221,99 @@ const SetupDXPCloudPage = ({
 					},
 				});
 
+				const currentDateAndTime = new Date().toUTCString();
+				const showDisasterDataCenterRegion = dxp?.disasterDataCenterRegion
+					? `Primary Disaster Center Region - ${dxp?.disasterDataCenterRegion}`
+					: ``;
+
+				const emailDataList = {
```
[the remark this answers is not in the document]
Comment (pedroliveirav) [reply modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/containers/setup-forms/SetupDXPCloudForm/index.jsx]: Done.

**Comment by Aline160** [comment]
Just starting reviewing :)

**Comment by pedroliveirav** [comment]
ci:test:sf

**Comment by Aline160** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent: https://github.com/pt-liferay-solutions/liferay-portal/pull/4393

**Pull request closed without merging**